### DISCUSSION TITLE
A dynamic tts approach

### DISCUSSION THREAD

**Discussion by jsjsam98** [open-webui/open-webui#4844] [General]
**Is your feature request related to a problem? Please describe.**
I noticed that current tts model is manually set on admin panel. But as a bilingual user, I would like the tts to be able to generate speech based on the output language. e.g.(I ask chat how to say Apple in Chinese, i would like to hear ‘apple’ pronounced in Chinese.

**Describe the solution you'd like**
online: google language detection 
local: https://github.com/pemistahl/lingua-py
**Describe alternatives you've considered**
User should select primary and secondary language, to make language detection process easier

**Additional context**
I can start work on this feature cuz it would be helpfull to me :)

**Comment by silentoplayz** [comment]
Related - https://github.com/open-webui/open-webui/issues/2256
